CHANGES.md

```diff
@@ -11,6 +11,7 @@
 - Updates use of deprecated options on createImageBitmap. [#12664](https://github.com/CesiumGS/cesium/pull/12664)
 - Fixed raymarching step size for cylindrical voxels. [#12681](https://github.com/CesiumGS/cesium/pull/12681)
 - Fixes handling of tileset `modelMatrix` changes for translations and rotations in `GaussianSplatPrimitive`. [#12706](https://github.com/CesiumGS/cesium/pull/12706)
+- Fixes an exception when removing a Gaussian splat asset from the scene primitives when it has more than one tile.
```

Comment:
The July release has gone out. This would go into into the next release on August 1.